kotlin-runtime/ftl-runtime/src/test/kotlin/xyz/block/ftl/schemaextractor/testdata/dependencies/BuiltinModuleClient.kt

```diff
@@ -0,0 +1,33 @@
+// Code generated by FTL-Generator, do not edit.
```

Comment:
added this `testdata/dependencies` directory which gets loaded up and added to the classpath for the detekt tests with type resolution. we can use it to add generated sources like`ftl.builtin` and `ftl.time`, ensuring these types get resolved in tests